docs/using/tutorial/Vagrantfile.template

```diff
@@ -10,6 +10,332 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+# To keep tutorial users from having to download several files, we will include
+# raw dumps of the node-side certificates here.
+
+clustercrt = <<EOL
```

Comment:
Rather than have the keys here in the vagrant file, and loaded from the other vagrant file, would it make sense to put all these files in the base image, and then just copy the appropriate node files into place on each node?